easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017a-mpi.eb

```diff
@@ -7,20 +7,22 @@ description = """NAMD is a parallel molecular dynamics code designed for high-pe
  large biomolecular systems."""
 
 toolchain = {'name': 'foss', 'version': '2017a'}
-toolchainopts = {'usempi': True, 'pic': True}
+toolchainopts = {'pic': True, 'usempi': True, 'openmp': True}
 
 sources = ['NAMD_%(version)s_Source.tar.gz']
 
+patches = [
+    'charm-remove-auto-from-fortran-args.patch',
+    'NAMD-%(version)s-drop_space_after_backslash.patch'
+]
+
 dependencies = [
-    ('Tcl', '8.5.19'),
-    ('FFTW', '3.3.6'),
+    ('Tcl', '8.6.6'),
```

Comment:
@henkela Thoughts on this?

You originally contributed this in #4516 with Tcl 8.5.19, but thanks to the enhancements to the NAMD easyblock by @akesandgren in https://github.com/hpcugent/easybuild-easyblocks/pull/1173, Tcl versions other than 8.5.x are also supported.

We usually don't fiddle with dependency versions after an easyconfig got merged, but I'm inclined to make an exception here...